lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _get = require('lodash.get');
+
+exports.register = (defaultRate, routes) => {
+  const rateTable = {};
+
+  routes.forEach((route) => {
```

Comment:
`reduce` is typically used as an aggregation function, not sure if design-wise it's the best way to go (even in terms of readability `forEach` is a little more clear)